pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1044,8 +1044,11 @@ public Schema getTableSchema(String tableName) {
   public void addTable(TableConfig tableConfig)
       throws IOException {
     if (isSingleTenantCluster()) {
```

Comment:
Not sure if we should perform this check. Maybe we should always do this null check?

Reply:
true. Removed isSingleTenantCluster check